templates/module/helfi_tpr/tpr-service--teaser-search-result.html.twig

```diff
@@ -1,5 +1,12 @@
+{%
+  set card_tpr_classes = [
+  not service_published ? 'card--unpublished',
```

Comment:
This variable could be more generic. Like `not content_entity_published ? 'card--unpublished',`

Reply:
Made it generic 🦖